src/Operation/Aggregate.php

```diff
@@ -276,6 +280,10 @@ private function createCommand(Server $server, $isCursorSupported)
             $cmd['bypassDocumentValidation'] = $this->options['bypassDocumentValidation'];
         }
 
+        if (isset($this->options['comment'])) {
+            $cmd['comment'] = $this->options['comment'];
+        }
+
```

Comment:
Since `comment` and `maxTimeMS` are the same in the sense that we want to check if they're `isset()` and then copy them as-is into the `$cmd` document, I propose creating a `foreach` as we have in other command helpers:

```php
foreach (['comment', 'maxTimeMS'] as $option) {
    if (isset($this->options[$option])) {
        $cmd[$option] = $this->options[$option];
    }
}
```

`allowDiskUse` is always set, so we can leave its earlier assignment as-is.